### PR TITLE
Bump up version to 0.19.0

### PR DESCRIPTION
**Pull request by AhdraMeraliQB**
## Description

image

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -51,6 +51,6 @@ authors:
 - family-names: Theisen
   given-names: Merel
 title: Kedro
-version: 0.18.14
-date-released: 2023-10-18
+version: 0.19.0
+date-released: 2023-12-12
```
Comment (AhdraMeraliQB) [inline CITATION.cff:55]: 2023-12-12* for any Americans

**Review by astrojuanlu — APPROVED** [review]
Just send it

**Review by merelcht — APPROVED** [review]
image

**Review by ankatiyar — APPROVED** [review]
giphy (1)

**Comment by astrojuanlu** [comment]
```
...
Collecting kedro-telemetry~=0.2.0 (from -r requirements.txt (line 9))
  Using cached kedro_telemetry-0.2.5-py3-none-any.whl (8.5 kB)
...
INFO: pip is looking at multiple versions of kedro-telemetry to determine which version is compatible with other requirements. This could take a while.
Collecting kedro-datasets[pandas.CSVDataset] (from -r requirements.txt (line 8))
  Using cached kedro_datasets-2.0.0-py3-none-any.whl (144 kB)
  Using cached kedro_datasets-1.8.0-py3-none-any.whl (151 kB)
  Downloading kedro_datasets-1.7.1-py3-none-any.whl (139 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 140.0/140.0 kB 5.6 MB/s eta 0:00:00
  Downloading kedro_datasets-1.7.0-py3-none-any.whl (130 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 130.7/130.7 kB 10.9 MB/s eta 0:00:00
  Downloading kedro_datasets-1.6.0-py3-none-any.whl (128 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 128.0/128.0 kB 11.0 MB/s eta 0:00:00

The conflict is caused by:
    The user requested kedro~=0.19.0
    kedro-datasets[pandas-csvdataset] 1.6.0 depends on kedro>=0.16
    kedro-telemetry 0.2.5 depends on kedro~=0.18.0
...
```

**Pull request merged**